css-three-column-layout/index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Three Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <main class="container">
+    <div class="row">
+      <h1 class="align-text-center">Designed for life science. Optimized for SaMD and Digital Health.</h1>
+    </div>
+    <div class="row">
+      <div class="column-one-third">
+        <div class="column-full">
+          <div class="image-div">
+            <img src="images/merge.png" alt="Merge">
+          </div>
+        </div>
+        <h2 class="align-text-center">Streamlined QMS</h2>
+        <p class="align-text-center">A QMS that includes modules for all stages of the product life cycle
+          from Design Control to CAPA.
+        </p>
+      </div>
```

Comment:
Rather than nesting an additional div to do your centering, try adding a second class to your column like this

```html
<div class="column-one-third justify-center">
  <!-- image, heading, paragraph -->
</div>
```

```css
.justify-center {
  display: flex;
  justify-content: center;
}
```